plugins/pubIds/urn/templates/settingsForm.tpl

```diff
@@ -0,0 +1,99 @@
+{**
+ * plugins/pubIds/urn/templates/settingsForm.tpl
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * URN plugin settings
+ *
+ *}
+
+<div id="description">{translate key="plugins.pubIds.urn.manager.settings.description"}</div>
+
+<script src="{$baseUrl}/plugins/pubIds/urn/js/URNSettingsFormHandler.js"></script>
```

Comment:
Could you try using Nate's recent `TemplateManager::addJavaScript` call here? I think this will be a new use case for it, but worth trying.